webtoolbar.py

```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
         selection = self._search_view.get_selection()
         model, selected = selection.get_selected()
 
+        if event.get_state() & Gdk.ModifierType.CONTROL_MASK:
+            if keyname == 'Return':
+                m = re.search('\.', self.props.text)
+                if self.props.text == "":
+                    return False
+                if m is None:
+                    self._set_text("www."+self.props.text+".com")
```

Comment:
1.  I don't think we need to add `www.`.  Websites work fine without it and we don't want to ruin any subdomains the user has entered.

2.  Lets take some ideas from the iOS keyboard.  If the user has typed `google` we should add `.com` (as you already do).  But if the user has typed `google.` we should only add `com`.